vendor/github.com/containers/storage/store.go

```diff
@@ -1109,6 +1110,13 @@ func (s *store) getROLayerStoresLocked() ([]roLayerStore, error) {
 		return nil, err
 	}
 	for _, store := range s.graphDriver.AdditionalImageStores() {
+		if strings.Contains(store, "|") {
+			parts := strings.Split(store, "|")
+			if parts[1] == "NO_REUSE" {
+				continue
+			}
+			store = strings.Split(store, "|")[0]
+		}
```

Comment:
I can’t see how this would work; layers in these stores would be effectively invisible, but images in those stores would be visible as usual.

The immediate effect is that all images from a no-reuse store are visible but have unresolvable layers = look broken to all existing tooling.